blockchyp/blockchyp_client.py

```diff
@@ -3,20 +3,23 @@
 """
 
 
+from datetime import datetime
```

Comment:
I generally separate my imports into categories, with an empty line in between: core library, third party, internal

ie:

```
from datetime import datetime

import requests

from blockchyp import CryptoUtils
```